### PR TITLE
Better SEO for the F# Guide

### PR DESCRIPTION
**Pull request by cartermp**
I'm assuming that these help with SEO for F#.  Is that accurate?

### CODE REVIEW

**Comment by mairaw** [comment]
Not really @cartermp. We're not even recommending adding them anymore. Apparently, Bing used to use them and Google doesn't use them since 2009. We should just remove them all. Title and URL are the biggest contributors to SEO.

**Comment by cartermp** [comment]
Sounds good.  I'll change wording instead.

File: docs/fsharp/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: F# Guide
-description: Learn about the F# programming language, an open-source language for .NET that provides first-class support for functional programming.
+description: Learn about F#, a functional programming language which runs on .NET.
```
Comment (mairaw) [inline docs/fsharp/index.md]: which runs -> that runs or add comma before which

File: docs/fsharp/index.md [hunk 2]
```diff
@@ -14,23 +14,21 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
-
-The [Tutorials](tutorials/getting-started/index.md) also have step-by-step guides for various skill levels and features of the language.
+If you're new to F#, begin with the [Tour of F#](tour.md) to get an overview of the language and some of its programming concepts.  If you're using Visual Studio, the Tutorial project template contains the same content.
 
 ## If You're Experienced with F# #
 
-If you know your way around F#, you'll find a lot of use in the [Language Reference](language-reference/index.md), which documents each aspect of the language thoroughly, supplemented by numerous code samples.  You'll also find a lot of use in the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference).  The F# Core Library Reference will eventually move away from MSDN and into these current docs.
+If you know your way around F#, or want to learn more about a specific language construct, you'll find a lot of use in the [Language Reference](language-reference/index.md).  It is a thorough guide of all F# language capabilities.
```
Comment (mairaw) [inline docs/fsharp/index.md]: you'll find a lot of use in the -> see the

It is -> It's

File: docs/fsharp/index.md [hunk 3]
```diff
@@ -14,23 +14,21 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+
+You'll also find a lot of use in the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference) when programming in F#.
```
Comment (mairaw) [inline docs/fsharp/index.md]: This sentence is also super informal. How about telling what they'll find in the reference?

Something like: When programming in F#, the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference)  is a great resource for ...

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks great. Left a couple of suggestions for you.

File: docs/fsharp/index.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: F# Guide
-description: Learn about the F# programming language, an open-source language for .NET that provides first-class support for functional programming.
+description: Learn about F#, a functional programming language which runs on .NET.
 keywords: .NET, .NET Core
 author: jackfoxy
 ms.author: phcart
```
Comment (mairaw) [inline docs/fsharp/index.md]: update ms.date too in this case

**Review by mairaw — APPROVED** [review]
LGTM. :shipit:

**Pull request merged**